src/Parser/Monk/Mistweaver/Modules/Spells/AzeriteTraits/UpliftedSpirits.js

```diff
@@ -54,6 +80,40 @@ class UpliftedSpirits extends Analyzer {
       this.healing += (healAmount - this.baseHeal);
     }
   }
+
+
+  statistic() {
+    return (
+      <StatisticBox
+        position={STATISTIC_ORDER.OPTIONAL(70)}
+        icon={<SpellIcon id={SPELLS.UPLIFTED_SPIRITS.id} />}
+        value={`${formatNumber(this.cooldownReductionUsed / 1000) || 0}`}
+        label={(
+          <dfn data-tip={`You wasted ${this.cooldownReductionWasted / 1000 || 0} seconds of cooldown reduction.`}
+          >
+            Revival Seconds Reduced
+          </dfn>
+        )}
+      />
+    );
+  }
+
+  subStatistic() {
+    return (
+      <div className="flex">
+        <div className="flex-main">
+          <SpellLink id={SPELLS.UPLIFTED_SPIRITS.id}>
+             Font of Life
+          </SpellLink>
```

Comment:
"Font of Life" instead of "Uplifted Spirits"

Reply:
Thanks for the catch. Also I have updated to use the new refactor for StatBoxItems.